cpp/test/CMakeLists.txt

```diff
@@ -138,44 +138,42 @@ add_executable(test_raft
 )
 
 set_target_properties(test_raft
-PROPERTIES BUILD_RPATH                         "\$ORIGIN"
-           # set target compile options
-           CXX_STANDARD                        17
-           CXX_STANDARD_REQUIRED               ON
-           CUDA_STANDARD                       17
-           CUDA_STANDARD_REQUIRED              ON
-           POSITION_INDEPENDENT_CODE           ON
-           INTERFACE_POSITION_INDEPENDENT_CODE ON
-           INSTALL_RPATH "\$ORIGIN/../../../lib"
-)
+        PROPERTIES BUILD_RPATH                         "\$ORIGIN"
+        # set target compile options
+        CXX_STANDARD                        17
+        CXX_STANDARD_REQUIRED               ON
+        CUDA_STANDARD                       17
+        CUDA_STANDARD_REQUIRED              ON
+        POSITION_INDEPENDENT_CODE           ON
+        INTERFACE_POSITION_INDEPENDENT_CODE ON
+        INSTALL_RPATH "\$ORIGIN/../../../lib"
+        )
 
 target_compile_options(test_raft
         PRIVATE "$<$<COMPILE_LANGUAGE:CXX>:${RAFT_CXX_FLAGS}>"
-                "$<$<COMPILE_LANGUAGE:CUDA>:${RAFT_CUDA_FLAGS}>"
-)
+        "$<$<COMPILE_LANGUAGE:CUDA>:${RAFT_CUDA_FLAGS}>"
+        )
 
 target_include_directories(test_raft
-    PUBLIC  "$<BUILD_INTERFACE:${RAFT_SOURCE_DIR}/test>"
-)
+        PUBLIC  "$<BUILD_INTERFACE:${RAFT_SOURCE_DIR}/test>"
+        )
 
 
 target_link_libraries(test_raft
-PRIVATE
-  raft::raft
-  raft::distance
-  raft::nn
-  NCCL::NCCL
-  faiss::faiss
-  GTest::gtest
-  GTest::gtest_main
-  Threads::Threads
-  $<TARGET_NAME_IF_EXISTS:OpenMP::OpenMP_CXX>
-  $<TARGET_NAME_IF_EXISTS:conda_env>
-)
+        PRIVATE
+        raft::raft
+        raft::distance
+        raft::nn
+        faiss::faiss
```

Comment:
Does this mean RAFT tests also require that FAISS is installed? I didn't see that mentioned in the `BUILD.md` or `README.md`. (I suppose the section on tests in `BUILD.md` is a good place to add this and other requirements for building and running tests)